app/interactors/projects/destroy.rb

```diff
@@ -6,8 +6,11 @@ class Destroy
 
     def call
       context.destroy
-
       context.fail!(error: "Error") unless project.destroy
+
+      if context.success?
```

Comment:
Why? Interactor doesn't reach this code it `context.fail!` was called.

Reply:
Also it is good practice separate logic to different interactors: destroying in one interactor, sending notification in another

Reply:
Or in this case you can code it like:
```suggestion
      end
      
      after do
        send_notification
      end
      
      private
      
      def send_notification
      ...
```